recipes/dspawpy/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "dspawpy" %}
+{% set version = "1.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f66646ad4f0eee568f6c638062cafa2bc12bfc58de4e627f728e30af8a8c6c4d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - h5py >=3.7
+    - pymatgen >=2021.2.8.1
+    - python >=3.6
+    - statsmodels >=0.12.0
+
+test:
+  imports:
+    - dspawpy
+    - dspawpy.analysis
+    - dspawpy.diffusion
+    - dspawpy.io
+
+about:
+  home: "http://www.hzwtech.com/"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

This should match the filename in the sdist:

```suggestion
  license_file: LICENSE.txt
```